### ISSUE TITLE
[Question] Entity visibility based on different entity (flexible solution)

### ISSUE THREAD

**Issue by VivantSenior** [thomasloven/lovelace-auto-entities#452]
Hi,

I'm looking for a way to display entities based on different entity but both entities need to belong to the same device. The solution would need to be flexible (using asterisks). Is it possible?

E.g:

First device's entities:
cover.office_left_window_blinds
binary_sensor.office_left_window_blinds_charging_status
sensor.office_left_window_blinds_battery

Second device's entities:
cover.bedroom_left_window_blinds
binary_sensor.bedroom_left_window_blinds_charging_status
sensor.bedroom_left_window_blinds_battery

Pseudocode:
```
if (binary_sensor.*_blinds_charging_status)
{
    show(sensor.*t_window_blinds_battery);
}
```

Basically, if a blind is charging show its battery level.
